main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,269 @@
+	 // Throws error
+	 purse.deposit(secondPayment, fungible123);
+     ```
+- [`purse.makeDepositFacet()`](https://agoric.com/documentation/ertp/api/purse.html#purse-makedepositfacet)
+  - Creates a deposit-only facet on the `purse` that can be given
+    to other parties. This lets them make a deposit, but not make
+    withdrawals, to your purse.
+  - ```js
+     const depositOnlyFacet = purse.makeDepositFacet();
+     // Give depositOnlyFacet to someone else. They can pass a payment
+     // that will be deposited:
+     depositOnlyFacet.receive(payment);
+     ```
+- [`purse.withdraw(amount)`](https://agoric.com/documentation/ertp/api/purse.html#purse-withdraw-amount)
+  - Withdraw the `amount` from this purse into a new
+    `payment`. Returns the new `payment`.
+
+In addition, other objects, such as `issuer`, have methods which take
+purses as arguments or return purses. In particular:
+- [`issuer.makeEmptyPurse()`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-makeemptypurse)
+  - Returns a new empty `purse` for storing an `amount` of the same `brand` as
+  the `issuer`.
+  - ```js
+	   const { issuer, mint } = produceIssuer('bucks');
+	   const purse = issuer.makeEmptyPurse();
+   ```
+
+## Payments
+
+![Payment methods](payment.svg)   
+
+Payments hold an `amount` of digital assets that are either about to
+be sent to another party or which is in transit to another
+party. Payments are linear, meaning that either a `payment` has its full
+original balance, or it is used up entirely. It is impossible to
+partially use a `payment`. In other words, if you create a `payment` with
+an `amount` of 10 quatloos, the `payment` will always either have an
+`amount` of 10 quatloos or an `amount` of 0 quatloos, never anything
+in-between or larger than 10.
+
+Payments can be deposited in purses, split into multiple 
+payments, combined, and claimed (getting an exclusive payment and
+revoking access from anyone else). 
+
+Payments are often received from other actors and therefore should not
+be trusted themselves. To get the balance of a payment, use the
+`getAmountOf(payment)` method on the trusted `issuer` for the `brand`
+of the `payment`.  **tyg todo: How does one get said trusted issuer?**
+
+To convert a `payment` into a `purse`: 
+1. Access a trusted `issuer` for the `payment`'s `brand`.. 
+2. Create an empty purse with `issuer.makeEmptyPurse()`
+3. Put the payment `amount` in the purse with `purse.deposit(payment)`
+
+Payments have one API method, but many methods for other object
+types have `payment` objects as arguments and effectively operate on `payments`.
+- [`payment.getAllegedBrand()`](https://agoric.com/documentation/ertp/api/payment.html#payment-getallegedbrand)
+  - Returns the `payment`'s `brand`, indicating what digital asset the
+  payment purports to be. Note that `payment`s are not trusted, so this
+  result should be treated with suspicion and verified before 
+  use. **tyg todo: How do I verify?**
```

Comment:
Any issuer operations on the payment, if they succeed, verify the payment. For example, `E(issuer).isLive(payment)`